pkg/alerts/allowed_conformance.go

```diff
@@ -2,61 +2,63 @@ package alerts
 
 import (
 	configv1 "github.com/openshift/api/config/v1"
-	helper "github.com/openshift/origin/test/extended/util/prometheus"
```

Comment:
I relocated the MetricCondition types to avoid a circular import when i started adding unit tests for the helper, which needed to access the allowed alerts in this package.